### PR TITLE
add labels in the benchmark and add kwargs to repeat

### PR DESCRIPTION
**Pull request by rushabh-v**

### CODE REVIEW

**Comment by rushabh-v** [comment]
Can you take a look, please? @grlee77

**Comment by emcastillo** [comment]
cc @asi1024

**Comment by grlee77** [comment]
Thanks @rushabh-v. It looks good to me.

File: cupyx/time.py [hunk 1]
```diff
@@ -33,14 +33,17 @@ def _to_str_per_item(t):
 
     def to_str(self, show_gpu=False):
         ts = self._ts if show_gpu else self._ts[[0]]
+        devices = ['CPU', 'GPU']
         return '{:<20s}:{}'.format(
-            self.name, ' '.join([self._to_str_per_item(t) for t in ts]))
+            self.name, ' '.join(["    " + devices[i] + ':'
+                                 + self._to_str_per_item(ts[i])
+                                 for i in range(len(ts))]))
```
Comment (asi1024) [inline cupyx/time.py]: How about adding `device_name` parameter to `_to_str_per_item`?

Comment (rushabh-v) [reply cupyx/time.py]: Done.

Comment (rushabh-v) [reply cupyx/time.py]: tests are also passing.

Comment (asi1024) [reply cupyx/time.py]: How about writing as follows?
```py
return '{:<20s}:{}'.format(
    self.name,
    ' '.join([self._to_str_per_item(*p) for p in zip(ts, device_names)])
```

Comment (rushabh-v) [reply cupyx/time.py]: Done

Comment (asi1024) [reply cupyx/time.py]: Why did you change the indentation?

Comment (rushabh-v) [reply cupyx/time.py]: It's changed by `autopep8`

Comment (rushabh-v) [reply cupyx/time.py]: Because the length of the last line becomes (80>79) there.

Comment (asi1024) [reply cupyx/time.py]: Please merge [this PR](https://github.com/rushabh-v/cupy/pull/1) for readability.

Comment (rushabh-v) [reply cupyx/time.py]: Merged!
Thanks. That looks much clearer now😄

**Comment by asi1024** [comment]
Jenkins, test this please.

**Comment by asi1024** [comment]
LGTM. Thanks for your PR!

**Pull request merged**